### PR TITLE
Armada file

### LINKED ISSUE #15: Create python file for handling all direct armada-client intergration
This file will handle:

- Submitting armada-jupyter podspec
- Any other armada-client intergrations
- Getting the token and URL from armada_jupyter pods. (Taken from #17 and #18)

I think under the unique circumstances it makes sense to do these at once, assuming I properly unit test them.

### PR DESCRIPTION
**Pull request by Sharpz7**
Closes #15

### CODE REVIEW

File: armada_jupyter/armada.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+from armada_jupyter.constants import HOST, PORT, DISABLE_SSL
+
+
+def create_jupyter_service():
+    """
+    Create a jupyter service.
+    """
+
+    return submit_pb2.ServiceConfig(
```
Comment (kannon92) [inline armada_jupyter/armada.py:18]: We have an issue to make these more configurable also?

File: armada_jupyter/armada.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        )
+
+    client = ArmadaClient(channel)
+
+    queue = submission.armada_queue
+    job_set_id = "armada-jupyter-pods"
+
+    # Create the PodSpec for the job
+    job_request_item = create_armada_request(submission, client)
+    client.submit_jobs(
```
Comment (kannon92) [inline armada_jupyter/armada.py:67]: When it comes to unit testing things like this, I like to have this logic refacted into a function.  

For your unit test, you could have client.submit_jobs be a dummy function so you could test this entire function minus the submit_jobs call.  

I'm okay with you holding off on that now but just a suggestion when you want to write more tests.

**Review by kannon92 — APPROVED** [review]
LGTM.  A few items we should track (configurable ingress/service) and allowing you to write unit tests of the submit function.

File: armada_jupyter/armada.py (hunk 1 above)
Comment (Sharpz7) [reply armada_jupyter/armada.py:18]: https://github.com/G-Research/armada-jupyter/issues/40

This was in the future - considerations in the design doc, so definitely not happening in the short term, but yes its definitely a needed feature.

File: armada_jupyter/armada.py (hunk 2 above)
Comment (Sharpz7) [reply armada_jupyter/armada.py:67]: Just for speed, maybe creating an issue for this and track it that way for now is okay?

**Pull request merged**